addons/mass_mailing/static/src/js/tours/mass_mailing_editor_tour.js

```diff
@@ -10,16 +10,41 @@ odoo.define('mass_mailing.mass_mailing_editor_tour', function (require) {
         trigger: '.o_app[data-menu-xmlid="mass_mailing.mass_mailing_menu_root"]',
     }, {
         trigger: 'button.o_list_button_add',
-    }, {
-        trigger: 'input[name="subject"]',
-        run: 'text Test',
     }, {
         trigger: 'div[name="contact_list_ids"] .o_input_dropdown > input[type="text"]',
     }, {
         trigger: 'li.ui-menu-item',
     }, {
-        trigger: 'iframe #default',
-        run: () => $('iframe').contents().find('o_editable').html('<p><br/></p>'),
+        trigger: '[name="body_arch"] iframe #empty',
+    }, {
+        trigger: '[name="body_arch"] iframe .o_editable'
+    }, {
+        trigger: '[name="body_arch"] iframe #email_designer_default_body [name="Title"] .ui-draggable-handle',
```

Comment:
Would you mind adding some explanation before your trigger to explain a bit what the step is about?

```js
// drag and drop a "body" snippet into the mail body editor
trigger: '[name="body_arch"] iframe #email_designer_default_body [name="Title"] .ui-draggable-handle',
```

(Not sure that's what it does, but it's to give an example).

Same for steps below unless they are crystal clear.